pages/index.js

```diff
@@ -0,0 +1,26 @@
+import JoinOurTeam from "../components/sections/JoinOurTeam";
+import OurWork from "../components/sections/OurWork";
+import Wrapper from "../components/wrapper/Wrapper";
+import Partners from "../components/sections/Partners";
+import WhoWeAre from "../components/sections/WhoWeAre";
+import WhatWeDo from "../components/sections/WhatWeDo";
+import ContactUs from "../components/sections/ContactUs";
+import Footer from "../components/layout/Footer";
+import Hero from "../components/sections/Hero";
+export default function Home() {
+  return (
+    
+    <>
```

Comment:
it's a good  practice when you are returning only one component like here to remove the fragment